packages/application/src/shell.ts

```diff
@@ -1629,7 +1629,9 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
     args: FocusTracker.IChangedArgs<Widget>
   ): void {
     if (args.newValue) {
-      args.newValue.title.className += ` ${CURRENT_CLASS}`;
+      args.newValue.title.className = `${args.newValue.title.className} ${CURRENT_CLASS} ${ACTIVE_CLASS}`;
```

Comment:
You will also need to remove the classes that you add because currently after witching back and forth this looks like this:
![Screenshot from 2023-02-03 23-01-18](https://user-images.githubusercontent.com/5832902/216727072-6e53f806-6af3-465b-bb6f-8037fb63a97a.png)

Reply:
That's a good point. I thought about this but decided to stay in line with existing implementation. Will update shortly

Reply:
It would also need a unit test in `jupyterlab/packages/application/test/shell.spec.ts` checking for the classes and that only one class is applied after switching for the second time as this logic is non-trivial (I had a strong suspicion that this will not be removed, but needed to check).

Reply:
Have updated code to remove active css class from old widget onCurrent changed. 

Having issues with running tests locally, need to solve this to be able to test unit tests
- #13938

Reply:
Added test that tests f current, active class names are cleared when current widget changes

Reply:
Thank you for pointing this out